git-imerge

```diff
@@ -3927,12 +3971,21 @@ def main(args):
 
     subparser = subparsers.add_parser(
         'reparent',
-        help='change the parents of the HEAD commit',
+        help=(
+            'change the parents of the specified commit and propagate the change to HEAD commit'
+            ),
         )
     subparser.add_argument(
-        'parents', nargs='*', help='[PARENT...]',
+        'parents', 
+        nargs='*', 
+        help=(
+            '[PARENT...] \n'
+            'parents is a list of SHA1.\n'
+            ),
```

Comment:
Newline characters don't have an effect in help strings. And instead of starting the help string with `[PARENT...]`, you can set `metavar='PARENT'`.